system_setup/server/controllers/locale.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from system_setup.common.helpers import get_windows_locale
+from subiquity.server.controllers.locale import LocaleController
+
+
+class LocaleController(LocaleController):
```

Comment:
Nitpick: I would call that WSLLocaleController as we are using the windows function (same for file name)

Reply:
Previously named WSLLocaleController but it end up not working

Reply:
Can you expand a little bit on this? We have definitively renamed controllers multiple times over the code in other parts and it works. What ended up not working? Have you tried debugging it?

Reply:
Sorry my bad, it seems it works now, I will change it to WSLLocaleController